package.json

```diff
@@ -76,6 +76,7 @@
     "@nestjs/jwt": "^10.0.2",
     "@nestjs/passport": "^9.0.3",
     "@nestjs/platform-express": "^9.0.0",
+    "@node-rs/xxhash-linux-x64-gnu": "^1.3.0",
```

Comment:
is this necessary for this PR?

Reply:
Is neccesary for run in linux. The server by regular is in linux